middleware/2fa.js

```diff
@@ -87,9 +87,8 @@ const sendCode = async (ctx, method, twoFactorMethod, requestId = -1, accountId
             }
             requestDetails += '<br/>';
         });
-        
     }
-    let subject = `Confirm Transaction with ${ receiver_id }`;
+    let subject = `Confirm Transaction from: ${ accountId }${ request ? ` to: ${ request.receiver_id }` : ``}`;
```

Comment:
@mattlockyer when do we have `request == null`? Shouldn't it always be available for 2FA?

Reply:
If the account is being set up and we are confirming the 2fa method with a code (there is no request) then requestId is -1 and request will be null.

Reply:
> If the account is being set up and we are confirming the 2fa method with a code (there is no request) then requestId is -1 and request will be null.
 
@mattlockyer hmm, what transaction we are confirming in that case? Deploying 2FA contract?